libuavcan_drivers/stm32/driver/src/uc_stm32_can.cpp

```diff
@@ -69,6 +69,8 @@ static int can2_irq(const int irq, void*);
 #define CAN1_RX1_IRQHandler    CAN_RX1_IRQHandler
 #endif
 
+#define STM32_CAN_RIR_RTR                          (1U << 1)            // Bit 1: Remote Transmission Request
+#define STM32_CAN_RIR_IDE                          (1U << 2)            // Bit 2: Identifier Extension
```

Comment:
Why do we need this here? These bits are already defined in bxcan.hpp, here: https://github.com/EShamaev/libuavcan/blob/f064efa80dba9ff125d7fded8c9f63bc3a4639bf/libuavcan_drivers/stm32/driver/include/uavcan_stm32/bxcan.hpp#L242-L243